packages/valory/skills/transaction_settlement_abci/rounds.py

```diff
@@ -477,9 +496,10 @@ class SynchronizeLateMessagesRound(CollectNonEmptyUntilThresholdRound):
     payload_class = SynchronizeLateMessagesPayload
     synchronized_data_class = SynchronizedData
     done_event = Event.DONE
-    no_majority_event = Event.NO_MAJORITY
     none_event = Event.NONE
-    collection_key = get_name(SynchronizedData.late_arriving_tx_hashes)
+    required_block_confirmations = 3
```

Comment:
@Adamantios review this carefully. We could be in a scenario where first k agents submit `None` and then we already transition when we are about to receive a non-`None` payload from `n-k`. Best we can do is wait a little

Reply:
`CollectNonEmptyUntilThresholdRound -> "This class represents logic for rounds where a round needs to collect
    optionally different payloads from k of n agents, where we only keep the non-empty attributes."`

So if the first k send `None` then we only go with `None`. But if say the k+1th would have sent a non-empty value then we'd be never seeing this. With zero block confirmations that's more likely than with non-zero...